README.md

```diff
@@ -1,2 +1,93 @@
-# FileOnQ.Imaging.Raw 
+# FileOnQ.Imaging.Raw
 
+**This project is a work in progress all code is currently experimental**
+
+A C#/.NET wrapper around [LibRaw](https://github.com/LibRaw/LibRaw) to simplify opening raw images and retrieving thumbnails.
```

Comment:
I know this may sound nitpicky but the technicality here is very important. This library doesn't open raw images, it opens thumbnails. The raw image is a digital negative which includes the raw pixel data which isn't readable by most image software. LibRaw processes the image so it can be viewed. If you have an unprocessed raw the photo will look washed out or the colors won't be right.

I suggest we change this to something like this:
```suggestion
A C#/.NET wrapper around [LibRaw](https://github.com/LibRaw/LibRaw) to simplify processing raw images and retrieving thumbnails.
```